exasol/toolbox/nox/_documentation.py

```diff
@@ -88,6 +105,117 @@ def clean_docs(_session: Session) -> None:
         shutil.rmtree(docs_folder)
 
 
+@nox.session(name="docs:links", python=False)
+def docs_list_links(session: Session) -> None:
+    """List all the links within the documentation."""
+    ignore = [r".*"]
+    env = os.environ.copy()
+    env["SPHINX_EXTRA_LINKCHECK_IGNORES"] = ",".join(ignore)
+    with tempfile.TemporaryDirectory() as path:
+        tmpdir = Path(path)
+        sp = subprocess.run(
+            [
+                "poetry",
```

Comment:
For nox tasks, we do not need the `poetry run` included. Could you remove it for `docs:links`, `docs:links:checks`, `docs:build`, and `docs:multiversion`.

Additionally, could you move the contents of `_build_docs` into `docs:build` and remove the `_build_docs`  function? Same with `_build_multiversion_docs` and `docs:multiversion`.


```suggestion
```